Mage.Sets/src/mage/sets/eighthedition/MurderousBetrayal.java

```diff
@@ -0,0 +1,121 @@
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ *  ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.sets.eighthedition;
+
+import java.util.UUID;
+
+import mage.ObjectColor;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.Cost;
+import mage.abilities.costs.CostImpl;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.DestroyTargetEffect;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.Rarity;
+import mage.constants.Zone;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.mageobject.ColorPredicate;
+import mage.game.Game;
+import mage.players.Player;
+import mage.target.common.TargetCreaturePermanent;
+
+/**
+ *
+ * @author anonymous
+ */
+public class MurderousBetrayal extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("nonblack creature");
+
+    static {
+        filter.add(Predicates.not(new ColorPredicate(ObjectColor.BLACK)));
+    }
+
+    public MurderousBetrayal(UUID ownerId) {
+        super(ownerId, 147, "Murderous Betrayal", Rarity.RARE, new CardType[]{CardType.ENCHANTMENT}, "{B}{B}{B}");
+        this.expansionSetCode = "8ED";
+
+        // {B}{B}, Pay half your life, rounded up: Destroy target nonblack creature. It can't be regenerated.
+        Effect effect = new DestroyTargetEffect(true);
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, effect, new MurderousBetrayalCost());
+        ability.addCost(new ManaCostsImpl("{B}{B}"));
+        ability.addTarget(new TargetCreaturePermanent(filter));
+        this.addAbility(ability);
+    }
+
+    public MurderousBetrayal(final MurderousBetrayal card) {
+        super(card);
+    }
+
+    @Override
+    public MurderousBetrayal copy() {
+        return new MurderousBetrayal(this);
+    }
+}
+
+class MurderousBetrayalCost extends CostImpl {
+
+    MurderousBetrayalCost() {
+        this.text = "Pay half your life, rounded up";
+    }
+
+    MurderousBetrayalCost(MurderousBetrayalCost cost) {
+        super(cost);
+    }
+
+    @Override
+    public boolean canPay(Ability ability, UUID sourceId, UUID controllerId, Game game) {
+        Player controller = game.getPlayer(controllerId);
+        return controller != null && !(controller.getLife() > 0 && !controller.canPayLifeCost());
```

Comment:
While it's true, that you can't pay life without life, the problem here is a little bit different.
The cost is calculated as 0 life, so you can pay 0 life.
118.4. If a cost or effect allows a player to pay an amount of life **greater than 0**, the player may do so only if his or her life total is greater than or equal to the amount of the payment.

Look as example at the rulings of Lurking Evil
http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=8817